engines/wintermute/base/base_object.h

```diff
@@ -134,6 +145,25 @@ class BaseObject : public BaseScriptHolder {
 	};
 	bool _nonIntMouseEvents;
 
+#ifdef ENABLE_WME3D
+	Math::Angle _angle;
```

Comment:
Maintenance wise, perhaps it would make sense for some of these classes where a lot of extra variables get added, to define a separate class that we instantiate, so the 3D-stuff gets logically moved to a separate file altogether, and the impact on the non 3D-code is lowered.

Just a thought.